docs/dictionary/command/visual-effect.lcdoc

```diff
@@ -15,21 +15,21 @@ OS: mac, windows, linux, ios, android, html5
 Platforms: desktop, mobile
 
 Example:
-visual effect dissolve
+visual effect "dissolve"
 go next card
 
 Example:
 -- This handler in an object will cause to hide then show 
 --  with a visual effect when clicked.
 on mouseUp
   hide me
-  show me with visual effect iris close slow
+  show me with visual effect "iris" close slow
```

Comment:
Would you also be able to add Show and Hide to the references too within this edit? Thanks again, Sean.